queries/studies.sql

```diff
@@ -161,7 +162,7 @@ WITH total AS (SELECT t.lesson_id,
 SELECT total.lesson_id as id, p.id as parent_id
 FROM users.profiles p
          JOIN completed ON completed.profile_id = p.id
-         JOIN total ON total.lesson_id = completed.lesson_id
+         JOIN total ON total.lesson_id >= completed.lesson_id -- In case somethig has been archived later
```

Comment:
>= doesn't make sense here on the join or?
Wouldn't there be a lot of rows?

Reply:
Loooool I put this in a totally wrong place :D. Gimme a moment.

Reply:
Fixed.